### PR TITLE
terminal-stocks: init at 1.0.14

### LINKED ISSUE #211183: Packaging Request: terminal-stocks
**Project description**
Fetch stock information, market summary right in your terminal.

**Metadata**

* homepage URL: https://shashi.dev/terminal-stocks/#/
* source URL: https://github.com/shweshi/terminal-stocks
* license: mit
* linux

### PR DESCRIPTION
**Pull request by mislavzanic**
###### Description of changes

Added a stock price cli application called [terminal-stocks](https://github.com/shweshi/terminal-stocks).
Closes [#211183](https://github.com/NixOS/nixpkgs/issues/211183).

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/terminal-stocks/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{ lib, buildNpmPackage, fetchFromGitHub, nix-update-script }:
+
+buildNpmPackage rec {
+  pname = "terminal-stocks";
+  version = "v1.0.14";
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/terminal-stocks/default.nix:5]:
```suggestion
  version = "1.0.14";
```

File: pkgs/applications/misc/terminal-stocks/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  src = fetchFromGitHub {
+    owner = "shweshi";
+    repo = pname;
+    rev = version;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/terminal-stocks/default.nix:10]:
```suggestion
    rev = "v${version}";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Review by aaschmid — APPROVED** [review]

**Comment by andriy-git** [comment]
Can't seem to find terminal-stocks package on the website or install it with: `nix-env -iA nixpkgs.terminal-stocks`

**Comment by mislavzanic** [comment]
The package will show up once it's merged into the nixos-unstable branch.

**Pull request merged**